server/serializers/project.go

```diff
@@ -1,8 +1,37 @@
 package serializers
 
+import (
+	"encoding/json"
+	"errors"
+	"io"
+
+	"github.com/Brightscout/mattermost-plugin-azure-devops/server/constants"
+)
+
 type ProjectDetails struct {
 	MattermostUserID string `json:"mattermostUserID"`
```

Comment:
Is mattermostuserID optional?

Reply:
yes, as we extract it from the header whenever we require it

Reply:
okay